packages/api/src/DAPI.js

```diff
@@ -23,6 +23,27 @@ class DAPI {
     const { epochsInfo } = await this.dapi.platform.getEpochsInfo(start, count, { ascending })
     return epochsInfo
   }
+
+  async getContestedState (contractId,
+    documentTypeName,
+    indexName = 'parentNameAndLabel',
+    resultType = 2,
+    indexValuesList, startAtIdentifierInfo,
+    allowIncludeLockedAndAbstainingVoteTally,
+    count
+  ) {
+    const { contestedResourceContenders } = await this.dapi.platform.getContestedResourceVoteState(
+      Buffer.from(contractId, 'base64'),
+      documentTypeName,
+      indexName,
+      resultType,
+      indexValuesList,
+      startAtIdentifierInfo,
```

Comment:
name params as in Dash SDK's getContestedResourceVoteState

Reply:
already